### PR TITLE
Fixing conversation tests

### PR DESCRIPTION
**Pull request by vytis**

### CODE REVIEW

File: Tests/Source/Integration/ConversationsTests.m [hunk 1]
```diff
@@ -898,24 +896,22 @@ @implementation ConversationTests (ConversationStatusAndOrder)
...
         [mockExtraConversation changeNameByUser:self.selfUser name:@"Extra conversation"];
     }];
     WaitForAllGroupsToBeEmpty(0.5);
     
-    [self.mockTransportSession performRemoteChanges:^(MockTransportSession<MockTransportSessionObjectCreation> *session) {
-        NOT_USED(session);
-        [self setDate:[NSDate dateWithTimeInterval:1000 sinceDate:self.groupConversation.lastEventTime] forAllEventsInMockConversation:mockExtraConversation];
-    }];
-    
+//    [self.mockTransportSession performRemoteChanges:^(MockTransportSession<MockTransportSessionObjectCreation> *session) {
```
Comment (typfel) [inline Tests/Source/Integration/ConversationsTests.m]: To be deleted?

File: Tests/Source/Integration/ConversationsTests.m [hunk 2]
```diff
@@ -326,8 +326,8 @@ @implementation ConversationTests (Participants)
 - (void)testThatParticipantsAreAddedToAConversationWhenTheyAreAddedRemotely
 {
     // given
-    XCTAssertTrue([self logInAndWaitForSyncToBeComplete]);
-    WaitForEverythingToBeDone();
+    XCTAssertTrue([self login]);
+    WaitForAllGroupsToBeEmpty(0.5);
```
Comment (daehn) [inline Tests/Source/Integration/ConversationsTests.m]: Can go?

File: Tests/Source/Integration/ConversationsTests.m [hunk 3]
```diff
@@ -347,8 +347,8 @@ - (void)testThatParticipantsAreAddedToAConversationWhenTheyAreAddedRemotely
 - (void)testThatParticipantsAreRemovedFromAConversationWhenTheyAreRemovedRemotely
 {
     // given
-    XCTAssertTrue([self logInAndWaitForSyncToBeComplete]);
-    WaitForEverythingToBeDone();
+    XCTAssertTrue([self login]);
+    WaitForAllGroupsToBeEmpty(0.5);
```
Comment (daehn) [inline Tests/Source/Integration/ConversationsTests.m]: Same here

File: Tests/Source/Integration/ConversationsTests.m [hunk 4]
```diff
@@ -262,16 +262,16 @@ - (void)testThatAddingParticipantsToAConversationIsSynchronizedWithBackend
...
     // Tears down context(s) &
     // Re-create contexts
-    [self recreateUserSessionAndWipeCache:YES];
-    XCTAssert([self logInAndWaitForSyncToBeComplete]);
+    [self recreateSessionManagerAndDeleteLocalData];
+    XCTAssert([self login]);
     
     // Wait for sync to be done
-    WaitForEverythingToBeDone();
+    WaitForAllGroupsToBeEmpty(0.5);
```
Comment (daehn) [inline Tests/Source/Integration/ConversationsTests.m]: Can be deleted

File: Tests/Source/Integration/ConversationsTests.m [hunk 5]
```diff
@@ -640,8 +639,8 @@ - (NSArray *)movedIndexPairsForChangeSet:(ConversationListChangeInfo *)note
 - (void)testThatNotificationsAreReceivedWhenConversationsAreFaulted
 {
     // given
-    XCTAssertTrue([self logInAndWaitForSyncToBeComplete]);
-    WaitForEverythingToBeDone();
+    XCTAssertTrue([self login]);
+    WaitForAllGroupsToBeEmpty(0.5);
```
Comment (daehn) [inline Tests/Source/Integration/ConversationsTests.m]: Can be deleted

File: Tests/Source/Integration/ConversationsTests.m [hunk 6]
```diff
@@ -683,13 +682,12 @@ - (void)testThatSelfUserSeesConversationWhenItIsAddedToConversationByOtherUser
 {
     // given
     
-    XCTAssertTrue([self logInAndWaitForSyncToBeComplete]);
-    WaitForEverythingToBeDone();
+    XCTAssertTrue([self login]);
+    WaitForAllGroupsToBeEmpty(0.5);
```
Comment (daehn) [inline Tests/Source/Integration/ConversationsTests.m]: Can be deleted

**Comment by vytis** [comment]
@daehn I think I removed all of them!

**Pull request merged**